lib/rules/restrict-full-import.js

```diff
@@ -1,66 +1,76 @@
 module.exports = {
   meta: {
     docs: {},
-    schema: [{
-      restricted: {
-        type: 'array',
-        items: {
-          type: 'string',
+    schema: [
+      {
+        restricted: {
+          type: 'array',
+          items: {
+            type: 'string',
+          },
+          uniqueItems: true,
         },
-        uniqueItems: true,
       },
-    }],
+    ],
   },
 
-  create: function(context) {
-    var restricted = context.options[0] || [];
+  create(context) {
+    const restricted = context.options[0] || [];
 
     function isRestrictedModule(mod) {
       return restricted.indexOf(mod) >= 0;
     }
 
+    /* eslint-disable prettier/prettier*/
```

Comment:
`eslint --fix` with the prettier/prettier config breaks this. It outputs invalid code. Create a reproducible issue in issue in `eslint-plugin-prettier`.

Reply:
I can confirm that this is a bug. See https://github.com/prettier/eslint-config-prettier/issues/31

Reply:
Instead of disabling prettier here, you can run `eslint --fix`, manually correct the syntax error, and the `eslint --fix` again.

Reply:
I'd prefer leaving as is. I've commented a link to the issue for context.